<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThesisLottery</class>
 <widget class="QWidget" name="ThesisLottery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>201</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThesisLottery</string>
  </property>
  <widget class="QSpinBox" name="thesisNumber_spinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumberOfIndividuals_spinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="thesisNumber_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Thesis number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NumberOfIndividuals_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of individuals:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="options_Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
